src/main/java/uk/gov/hmcts/reform/judicialbooking/domain/service/common/ParseRequestService.java

```diff
@@ -42,4 +45,15 @@ public BookingEntity parseBookingRequest(BookingRequest bookingRequest) throws P
         return ValidationUtil.validateBookingRequest(booking);
     }
 
+    public List<String> parseQueryRequest(BookingQueryRequest queryRequest) throws ParseException {
+        if (ObjectUtils.isEmpty(queryRequest.getQueryRequest().getUserIds())) {
+            throw new BadRequestException("Provided list of userIds is empty");
+        }
+
+        queryRequest.getQueryRequest().getUserIds().forEach(
+            userId -> ValidationUtil.validateInputParams(Constants.UUID_PATTERN, userId));
+
```

Comment:
ValidationUtil.validateInputParams van take array of userIds. so please use that instead of foreach.